src/Main.java

```diff
@@ -1,172 +1,100 @@
+        int agePeople = 25;
+        if (agePeople >= 2 && agePeople <= 6) {
+            System.out.println("Если возраст человека равен " + agePeople + " то ему нужно ходить в детский сад");
+        }
+        if (agePeople >= 7 && agePeople <= 17) {
+            System.out.println("Если возраст человека равен " + agePeople + " то ему нужно ходить в школу");
+        }
+        if (agePeople >= 18 && agePeople <= 24) {
+            System.out.println("Если возраст человека равен " + agePeople + " то ему нужно ходить в университет");
+        }
+        if (agePeople > 24) {
+            System.out.println("Если возраст человека равен " + agePeople + " то ему нужно ходить на работу");
+        }
 
         //Задача №5
-        byte whitePaint = 2;
-        System.out.println("Количество банок белой краски для 1 класса" + whitePaint);
-        byte drownPaint = 4;
-        System.out.println("Количество банок коричневой краски для 1 класса" + drownPaint);
-        byte allPaint = 120;
-        System.out.println("На ремонт школы закуплено" + allPaint + " банок краски");
-        short classPaint = (short) (whitePaint + drownPaint);
-        System.out.println("Количество банок для покраски 1 класса" + classPaint);
-        short allClasses = (short) (allPaint / classPaint);
-        System.out.println("В школе всего " + allClasses + "классов");
-        short allWhitePaint = (short) (whitePaint * allClasses);
-        System.out.println("Необходимо" + allWhitePaint + "банок белой краски для ремонта школы");
-        short allDrownPaint = (short) (drownPaint * allClasses);
-        System.out.println("Необходимо" + allDrownPaint + "банок коричневной краски для ремонта школы");
-        System.out.println("В школе, где" + allClasses + "классов, нужно" + allWhitePaint + "банок белой краски и" + allDrownPaint + "банок кричневой краски");
-
-        //Задача 6
-        short banana = 5;
-        System.out.println(banana);
-        short milk = 200;
-        System.out.println(milk);
-        short iceCream = 100;
-        System.out.println(iceCream);
-        short egg = 70;
-        System.out.println(egg);
-        short veightBanana = 5 * 80;
-        System.out.println("Вес пяти бананов составляет" + veightBanana + " гр ");
-        short veightMilk = 105 * 2;
-        System.out.println("Вес 200мл молока составляет" + veightMilk + " гр ");
-        short veightIceCream = 100 * 2;
-        System.out.println("Вес мороженого составляет" + veightIceCream + " гр ");
-        short veightEgg = 70 * 4;
-        System.out.println("Вес четырёх яиц составляет" + veightEgg + " гр ");
-        short veightBreakfast = (short) (veightBanana + veightMilk + veightIceCream + veightEgg);
-        System.out.println("Общий вес завтрака составлет" + veightBreakfast + " гр ");
-        float breakfast = veightBreakfast / 1000F;
-        System.out.println("Вес завтрака составляет" + breakfast + " кг ");
+        int ageChild = 15;
```

Comment:
можно было не создавать новую переменную, а просто использовать уже созданную
```suggestion
        age = 15;
```